### PR TITLE
Remove "bc-" prefix of barclamp id's

### PR DESCRIPTION
**Pull request by vwallfahrer**

### CODE REVIEW

File: chef/cookbooks/swift/recipes/test.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 t = <<END
 {
-  "id": "bc-template-swift",
+  "id": "Xtemplate-swift",
```
Comment (jsuchome) [inline chef/cookbooks/swift/recipes/test.rb]: typo

**Comment by tboerger** [comment]
Do we need some kind of migrations for already existing installations? Otherwise :+1:

**Comment by rsalevsky** [comment]
+1

**Comment by vuntz** [comment]
Can you rebase?

**Comment by vuntz** [comment]
This was not correctly rebased, as the nova_dashboard bits are not renamed to horizon here. Can you fix that?

**Comment by vuntz** [comment]
+1

**Comment by toabctl** [comment]
+1

**Pull request merged**